buildsystem/codecompliance/authors.py

```diff
@@ -6,6 +7,45 @@
 
 import re
 
+def get_email_obfuscation_substitutions():
+    """
+    What tokens in email should be replaced with
+    """
+    return {
+    '.' : ' noot ',
```

Comment:
why `noot`? As non-native English speaker, that doesn't immediately bring `.` to my mind. I once saw `dawt` used, perhaps this could be used? Or is it already considered compromised?